PublicationsRouter/v2/sword-out/XWALK-EPrints.md

```diff
@@ -0,0 +1,139 @@
+# JPER Core Metadata to Dublin Core/RIOXX XML
+
+This document describes the crosswalk from the notification metadata fields received via the JPER API to the
+ XML format supplied to repositories via SWORDv2.
+ 
+The approach to the crosswalk is:
+
+* If there is a DC or DCTerms field to which the value can be written, do so
+* If there is also a RIOXX field which is different, do that one too
+
+This means that repositories which impelement one or other or both will have good changes of extracting useful
+metadata.
+
```

Comment:
Repositories which implement one or other or both will have good chances of extracting useful metadata. 

Also, what is "one or other or both" in this context? I assume we're talking about whether the repository implements the DC schema or RIOXX schema?